taxi/forms.py

```diff
@@ -50,3 +54,18 @@
         raise ValidationError("Last 5 characters should be digits")
 
     return license_number
+
+
+class ManufacturerSearchForm(forms.Form):
+    name = forms.CharField(max_length=200, label="Search by name")
+
+
+class CarSearchForm(forms.Form):
+    model = forms.CharField(max_length=200, label="Search by model")
+
+
+class DriverSearchForm(forms.Form):
+    license_number = forms.CharField(
+        max_length=200,
```

Comment:
The `max_length` of `license_number` in `DriverSearchForm` should match the `max_length` in `DriverCreationForm` and `DriverLicenseUpdateForm` to maintain consistency.